app/models/concerns/orchestration.rb

```diff
@@ -82,14 +90,22 @@ def record_conflicts
     @record_conflicts ||= []
   end
 
+  def skip_orchestration!
```

Comment:
I agree, just note that `importing_facts` disables only some parts of orchestration. I suppose that's what was meant by more generic.